### ISSUE TITLE
HeadlessInAppWebView called from WorkManager background task triggers NullPointerException on missing context

### ISSUE THREAD

**Issue by Ro-El** [pichillilorenzo/flutter_inappwebview#912]
- [x] I have read the [Get Started - Installation](https://inappwebview.dev/docs/get-started/installation/) section
- [x] I have read and done the [Get Started - Setup Android](https://inappwebview.dev/docs/get-started/setup-android/) section
- [x] I have read and done the [Get Started - Setup iOS](https://inappwebview.dev/docs/get-started/setup-ios/) section
- [x] I have already searched for the same problem

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |       2.2.2        |
| Plugin version       |     5.3.2     |
| Android version      |      11         |
| iOS version          |               |
| Xcode version        |               |

Device information: SM-G780F

## Description

**Expected behavior:** 
HeadlessInAppWebView does not require context to run.

**Current behavior:** 
NullPointerException when trying to retrieve context 'java.lang.Object android.content.Context.getSystemService(java.lang.String)'

## Steps to reproduce

1. Setup workmanager background task
2. Setup top-level headlessWebView e.g.

```
HeadlessInAppWebView` headlessWebView = new HeadlessInAppWebView(
  initialUrlRequest: URLRequest(url: Uri.parse("https://github.com/flutter")),
  onWebViewCreated: (controller) {
    debugPrint('HeadlessInAppWebView created!');
  },
  onConsoleMessage: (controller, consoleMessage) {
    debugPrint('Console Message: ${consoleMessage.message}');
  },
  onLoadStart: (controller, url) async {
    debugPrint('onLoadStart $url');
  },
  onLoadStop: (controller, url) async {
    debugPrint('onLoadStop $url');
  },
);
```

3. Perform headlessWebview.run() from workmanager task

## Stacktrace/Logcat
```
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): Failed to handle method call
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): **java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference**
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at com.pichillilorenzo.flutter_inappwebview.in_app_webview.FlutterWebView.<init>(FlutterWebView.java:51)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at com.pichillilorenzo.flutter_inappwebview.headless_in_app_webview.HeadlessInAppWebViewManager.run(HeadlessInAppWebViewManager.java:69)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at com.pichillilorenzo.flutter_inappwebview.headless_in_app_webview.HeadlessInAppWebViewManager.onMethodCall(HeadlessInAppWebViewManager.java:58)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:818)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at android.os.MessageQueue.next(MessageQueue.java:335)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at android.os.Looper.loop(Looper.java:206)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at android.app.ActivityThread.main(ActivityThread.java:8506)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
E/MethodChannel#com.pichillilorenzo/flutter_headless_inappwebview(21758): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
E/BackgroundWorker(21758): errorCode: error, errorMessage: Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference
I/WM-WorkerWrapper(21758): Worker result FAILURE for Work [ id=ae88348e-4fa3-4abc-bf58-23d13083f4f9, tags={ be.tramckrijte.workmanager.BackgroundWorker, currentWork } ]
```

**Comment by xiaoyaoking** [comment]
Did you solve it? I had the same problem. workmanager flutter_inappwebview not working

**Comment by pichillilorenzo** [comment]
Fixed with release `5.5.0+5`
